cmd/release-tools/README.md

```diff
@@ -0,0 +1,73 @@
+# release-tools
+
+## Overview
+
+The `release.sh` tool is used to perform the basic tag and release process
+for Clear Containers . It is designed to be run after a version bump was
+done in the file `VERSION` in Clear Containers repositories.
+
+By default, it will ensure:
+
+- A new release is greater than the last release github.
+
+- Wont allow create a new release with the same tag.
+
+
+Optionally, the go program called release.go can be used to interact and
+get more information about release.
+
+```
+go build -o  release
+#Get last runtime version
+./release status runtime
+#Get the next version bump for runtime repository
+./release status --next-bump runtime
+```
+
+### For full details
+
+Run:
+
+```
+$ ./checkcommits -h
```

Comment:
is `checkcommits` reachable from the directory of this readme? If not, maybe add full path?